src/plugins/intel_cpu/src/nodes/mkldnn_color_convert_node.cpp

```diff
@@ -0,0 +1,659 @@
+        blendWithMask(1, d);
+        blendWithMask(2, e);
+    };  // blend
+
+    auto colorConvert = [&](const reg_type & y_val, const reg_type & uv_val) {
+        uni_vshufps(u_val, uv_val, uv_val, even_mask);              // u_val = tmp[0,0,2,2,4,4,6,6]
+        uni_vshufps(v_val, uv_val, uv_val, odd_mask);               // v_val = tmp[1,1,3,3,5,5,7,7]
+
+        uni_vbroadcastss(tmp, ptr[consts + 0 * sizeof(float)]);     // tmp = [16.0f,16.0f,...]
+        uni_vsubps(c, y_val, tmp);                                  // c = y_val - tmp
+        uni_vbroadcastss(tmp, ptr[consts + 1 * sizeof(float)]);     // tmp = [128.f,128.f,...]
+        uni_vsubps(d, u_val, tmp);                                  // d = u_val - tmp
+        uni_vsubps(e, v_val, tmp);                                  // e = v_val - tmp
+
+        uni_vbroadcastss(tmp, ptr[consts + 2 * sizeof(float)]);     // tmp = [1.164f,1.164f,...]
+        uni_vmulps(c, c, tmp);                                      // c = c * tmp
+
+        uni_vbroadcastss(r, ptr[consts + 3 * sizeof(float)]);       // r = [1.596f,1.596f,...]
+        uni_vmulps(r, r, e);                                        // r = r * e
+        uni_vaddps(r, r, c);                                        // r = r + c
+
+        uni_vbroadcastss(g, ptr[consts + 4 * sizeof(float)]);       // g = [0.391f,0.391f,...]
+        uni_vmulps(g, g, d);                                        // g = g * d
+        uni_vsubps(g, c, g);                                        // g = c - g
+        uni_vbroadcastss(tmp, ptr[consts + 6 * sizeof(float)]);     // tmp = [0.813f,0.813f,...]
+        uni_vmulps(tmp, tmp, e);                                    // tmp = tmp * e
+        uni_vsubps(g, g, tmp);                                      // g = g - tmp
+
+        uni_vbroadcastss(b, ptr[consts + 5 * sizeof(float)]);       // b = [2.018f,2.018f,...]
+        uni_vmulps(b, b, d);                                        // b = b * d
+        uni_vaddps(b, b, c);                                        // b = b + c
+
+        // clip
+        uni_vxorps(c, c, c);
+        uni_vbroadcastss(d, ptr[consts + 7 * sizeof(float)]);
+
+        clip(r, c, d);
+        clip(g, c, d);
+        clip(b, c, d);
+
+        _if(colorFormat == 0)
+        ._then([&]{ blend(r, g, b); })
+        ._else([&]{ blend(b, g, r); });
+    };
+
+    const size_t reg_capacity_log = static_cast<size_t>(std::logb(reg_capacity));
+    const size_t step = reg_capacity * sizeof(T);
+
+    width >>= reg_capacity_log;
+
+    foreach(0, width, [&](const Reg64 & idx) {
+        load(y_val, y);
+        load(uv_val, uv);
+
+        colorConvert(y_val, uv_val);
+
+        store(dst, c);  dst += step;
+        store(dst, d);  dst += step;
+        store(dst, e);  dst += step;
+
+        y += step;
+        uv += step;
+    });
+
+    mov(width, argPtr(&Params::width));
+    width &= reg_capacity - 1;
+
+    _if(width != 0)
+    ._then([&] {
+        auto s = stack(3 * step);
```

Comment:
Still think we can hide stack related logic inside load/store routines just to avoid its duplication inside each kernel. However current logic shouldn't be changes in bounds of this PR.